### DISCUSSION TITLE
2.8版本报错（已经安装了.net环境）

### DISCUSSION THREAD

**Discussion by miku9964** [pjy612/SteamManifestCache#12] [Question / Answer]
![image](https://github.com/pjy612/SteamManifestCache/assets/92928921/bd6f8d48-bf83-4494-9a71-5f07a1ccbdb2)
不知道该怎么解决，昨天更新之前倒是一切正常

**Comment by pjy612** [comment]
已修复。重新下一次就应该可以了。

**Reply by miku9964** [reply]
thank you
